src/VCF_Globals.cpp

```diff
@@ -8,10 +8,13 @@
 #include "SharedFirmwareTypes.h"
 
 /* Interface and system data structs */
-VCFInterfaceData_s interface_data;
-VCFSystemData_s system_data;
-VCRSystemData_s vcr_system_data;
+VCFData_s vcf_data = {};
 
-/* ADC setup */
-MCP_ADC<channels_within_mcp_adc> adc_1 = MCP_ADC<channels_within_mcp_adc>(ADC1_CS);
-MCP_ADC<channels_within_mcp_adc> adc_2 = MCP_ADC<channels_within_mcp_adc>(ADC2_CS);
\ No newline at end of file
+uint16_t VCF_PORT = EthernetIPDefsInstance::instance().VCFData_port;
+
+/* Ethernet Constants */
+const IPAddress debug_ip(192, 168, 1, 31); // Computer receive IP
+const IPAddress default_VCF_ip(192, 168, 1, 30);
+const IPAddress default_dns(192, 168, 1, 1); 
+const IPAddress default_gateway(192, 168, 1, 1); 
+const IPAddress car_subnet(255, 255, 255, 0); 
```

Comment:
we should be referring to `EthernetIPDefsInstance::instance().default_VCF_ip` etc